test/test_torch.py

```diff
@@ -17836,49 +17836,68 @@ def test_case_info(fn_name, config):
                         f'Subprocess exception while attempting to run {test_case_info(fn_name, config)}:\n'
                         + e.output.decode("utf-8")))
 
-    @onlyCPU
+    @precisionOverride({torch.half: 0.05, torch.bfloat16: 0.05})
+    @onlyOnCPUAndCUDA
+    @dtypesIfCUDA(*(torch.testing.get_all_fp_dtypes(include_bfloat16=AMPERE_OR_ROCM)))
     @dtypes(*(torch.testing.get_all_complex_dtypes() + [torch.float, torch.double]))
+    @tf32_on_and_off(0.05)
```

Comment:
floating_and_complex_types() on the previous line

Reply:
fixed

Reply:
Is this commit missing?

Reply:
It's not missing, after offline discussion, we decided that instead of testing on `floating_and_complex_types()`, we test on all fp and complex, and assertRaise for unsupported dtypes. So this comment no longer applies.